llarp/util/encode.hpp

```diff
@@ -80,7 +80,8 @@ namespace llarp
   Base32Encode(const V& value, Stack& stack)
   {
     size_t ret = 0, pos = 1;
-    int bits = 8, tmp = value[0];
+    int bits = 8;
+    uint32_t tmp = value[0];
```

Comment:
why `uint32_t` ?

Reply:
encode.hpp:92:15: runtime error: left shift of 230014456 by 8 places cannot be represented in type 'int32_t' (aka 'int')
SUMMARY: UndefinedBehaviorSanitizer: undefined-behavior /Users/admin/Sites/llarp/llarp/util/encode.hpp:92:15 in 

something about shifted a signed 32 bit integer.